src/main/java/jose/task/Deadline.java

```diff
@@ -0,0 +1,51 @@
+package jose.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
```

Comment:
Perhaps a header comment for JavaDocs regarding description of the class is missing?

I see that some of the classes also have such mistakes.

Reply:
They are indeed missing. Thanks